Tools/CMake/AMReXParallelBackends.cmake

```diff
@@ -210,6 +210,10 @@ if (AMReX_HIP)
    find_package(rocrand REQUIRED CONFIG)
    find_package(rocprim REQUIRED CONFIG)
    find_package(hiprand REQUIRED CONFIG)
+   if (AMReX_ROCTX)
+   target_include_directories(amrex PUBLIC ${HIP_PATH}/../roctracer/include ${HIP_PATH}/../rocprofiler/include)
```

Comment:
I think this might be a tool we can `find_package()` for: https://github.com/ROCm-Developer-Tools/roctracer

Reply:
I tried something similar to the here: https://github.com/AMReX-Codes/amrex/pull/2057/commits/81230627bc8391f1f0e0c5f373312b569e11cdaa#diff-f817df7657a979c7f7e4013a878563aeaadfbbc09a12d5b60bbf697a04ef4ff4R212-R218

There's no cmake directory in roctracer/libs like there is in rocrand/libs. I couldn't find any obvious place for a cmake config file, and it didn't look like roctracer generates one. How do I find out what alias is appropriate?

Reply:
I get this error when I try to use findPackage on roctracer, with or without setting `roctracer_DIR`:

```CMake Error at subprojects/amrex/Tools/CMake/AMReXParallelBackends.cmake:214 (find_package):
  Could not find a package configuration file provided by "roctracer" with
  any of the following names:

    roctracerConfig.cmake
    roctracer-config.cmake

  Add the installation prefix of "roctracer" to CMAKE_PREFIX_PATH or set
  "roctracer_DIR" to a directory containing one of the above files.  If
  "roctracer" provides a separate development package or SDK, be sure it has
  been installed.```

Reply:
Ah I see, ok I submitted an issue so that we can modernize accordingly in the near future: https://github.com/ROCm-Developer-Tools/roctracer/issues/56